capstone_project/src/main/webapp/scripts/mapsScriptTemplate.js

```diff
@@ -18,7 +18,74 @@
 function createMap() { // eslint-disable-line no-unused-vars
   const map = new google.maps.Map(
       document.getElementById('map'),
-      {center: {lat: 39.828502, lng: -98.579512}, zoom: 4});
+      {center: {lat: 39.828502, lng: -98.579512},
```

Comment:
nit: change this object to block format to be consistent with the styles variables down below